wiki/templates/wiki/admin_panel.html

```diff
@@ -0,0 +1,33 @@
+{% extends "wiki/base.html" %}
+{% load i18n wiki_tags sekizai_tags %}
+
+{% block wiki_pagetitle %}{% trans "Admin Panel" %}{% endblock %}
+
+{% block wiki_contents %}
+
+<h1 class="page-header">{% trans "Deleted Articles" %}</h1>
+{% if deleted_articles %}
+  <table class="table table-striped">
+    <thead>
+      <tr>
+        <th>{% trans "Page Title" %}</th>
+        <th>{% trans "Date Deleted" %}</th>
+        <th>{% trans "Restore Article" %}</th>
+      </tr>
+    </thead>
+    <tbody>
+    {% for article in deleted_articles %}
+      <tr>
+        <td><a href="{{article.get_absolute_url}}">{{ article }}</a></td>
+        <td> {{article.modified}} </td>
+        <td><a href="{{article.get_absolute_url}}_deleted/?restore=1" class="btn btn-default"><span class="fa fa-repeat"></span>
```

Comment:
Sorry, I have one more comment: This should be an actual URL handled by an actual view -- for easy testability and maintainability of the project. The link should point to `{% url 'wiki:deleted' article_id=article.id %}?restore=1`. You should use the ID of the article, not the path because not all articles are guaranteed to have paths.

This would enable you to do a test case like

1. Create an article (already has test code you can use)
2. Delete it (already has test code you can use)
3. Go to admin panel (might wanna rename it to restore_list) and test that it's there
4. Visit `reverse('wiki:deleted', kwargs={'article_id': test_article.id}) + "?restore=1"` and check that it redirects to the article (I think it responds with a redirect?).